playpen/coverage/README.md

```diff
@@ -0,0 +1,60 @@
+# pulp coverage
+
+In this directory, there is a `sitecustomize.py` module, which turns on coverage tracking for all
+python interpreters started after its installation in a hosts's site-packages dir. Alongside that
+module is a script, `coverage_hook`, which is capable of installing and uninstalling the
+sitecustomize coverage hook, as well as a requirements file, which contains all the packages
+depended on by `coverage_hook` and `sitecustomize.py` that are no already provided by Pulp or any
+of its plugins.
+
+`coverage_hook` has several subcommands and options; consult that script's help for more detailed
+usage information. Of particular note, the 'report' subcommand allows for additionally generating
+html and xml report types.
+
+Because `coverage_hook` copies files to (and removes files from) python's site-wide packages dir,
+and must be able to read coverage data by any user, _root privileges are required_.
+
+## The workflow
+
+To generate a coverage report for Pulp usage, here are the steps to take:
+
+- Stop all running Pulp processes (including all workers and https WSGI daemons)
+- Install the coverage hook (`sudo ./coverage_hook install`)
+- Start up Pulp, workers/httpd/etc.
+- Use pulp (e.g. run the `pulp-smash` against it)
```

Comment:
s/pulp/Pulp/

I don't think we are consistent about this, but you seem to be elsewhere in this file.